dotcom-rendering/src/web/components/DynamicFast.tsx

```diff
@@ -19,39 +18,16 @@ export const DynamicFast = ({ trails, containerPalette, showAge }: Props) => {
 		<>
 			<UL direction="row" padBottom={true}>
 				<LI padSides={true} percentage="75%">
-					<Card
+					<FrontCard
+						trail={primary}
 						containerPalette={containerPalette}
 						showAge={showAge}
-						linkTo={primary.url}
-						format={primary.format}
-						headlineText={primary.headline}
-						trailText={primary.trailText}
+						//Overrides
```

Comment:
```suggestion
```

I tend to find comments like this that apply to multiple props below it often don't age well. Other changes come in later that mean the comment no longer applies or worse, is inaccurate.

Maybe we can do something here with JSDoc?